admin/README.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+
+## Build the `galene` binary
+
+You will need Go 1.13 or later (type `go version`).  Then do:
+
+	cd admin
+    CGO_ENABLED=0 go build -ldflags='-s -w'
```

Comment:
Please format your files, it seems that you have some tabulations done with `\t` and some with spaces.

For Markdown there are many tools online able to format it. For Golang, there is <https://golang.org/cmd/gofmt/>.